samples/nrf9160/aws_fota/src/main.c

```diff
@@ -434,36 +437,44 @@ static int fds_init(struct mqtt_client *c)
 	return 0;
 }
 
-/**@brief Configures modem to provide LTE link.
- *
- * Blocks until link is successfully established.
- */
-static void modem_configure(void)
+/**@brief Configures AT Command interface to the modem link. */
+static void at_configure(void)
 {
-#if defined(CONFIG_LTE_LINK_CONTROL)
-	BUILD_ASSERT_MSG(!IS_ENABLED(CONFIG_LTE_AUTO_INIT_AND_CONNECT),
-			"This sample does not support auto init and connect");
 	int err;
 
 	err = at_notif_init();
 	__ASSERT(err == 0, "AT Notify could not be initialized.");
 	err = at_cmd_init();
 	__ASSERT(err == 0, "AT CMD could not be established.");
-	printk("LTE Link Connecting ...\n");
-	err = lte_lc_init_and_connect();
-	__ASSERT(err == 0, "LTE link could not be established.");
-	printk("LTE Link Connected!\n");
-#endif
 }
 
 static void aws_fota_cb_handler(enum aws_fota_evt_id evt)
 {
+	int err;
+
 	switch (evt) {
 	case AWS_FOTA_EVT_DONE:
 		printk("AWS_FOTA_EVT_DONE, rebooting to apply update.\n");
 		do_reboot = true;
 		break;
 
+	case AWS_FOTA_EVT_ERASE_PENDING:
+		printk("AWS_FOTA_EVT_ERASE_PENDING, reboot or disconnect the "
+		       "LTE link\n");
+		err = lte_lc_offline();
+		if (err) {
+			printf("Error turning off the LTE link\n");
```

Comment:
`printf` -> `printk` here and line 474.

Reply:
Thanks for spotting it was an autocomplete error :+1: